public/sitemap.xml

```diff
@@ -1 +1 @@
-<?xml version="1.0" encoding="UTF-8"?><urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"><url><loc>https://proto.school</loc></url><url><loc>https://proto.school/events</loc></url><url><loc>https://proto.school/host</loc></url><url><loc>https://proto.school/build</loc></url><url><loc>https://proto.school/contribute</loc></url><url><loc>https://proto.school/news</loc></url><url><loc>https://proto.school/tutorials</loc></url><url><loc>https://proto.school/data-structures</loc></url><url><loc>https://proto.school/basics</loc></url><url><loc>https://proto.school/blog</loc></url><url><loc>https://proto.school/mutable-file-system</loc></url><url><loc>https://proto.school/regular-files-api</loc></url><url><loc>https://proto.school/anatomy-of-a-cid</loc></url><url><loc>https://proto.school/verifying-storage-on-filecoin</loc></url></urlset>
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8"?><urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"><url><loc>https://proto.school/</loc><lastmod>2020-08-06</lastmod><changefreq>weekly</changefreq><priority>1.0</priority></url><url><loc>https://proto.school/tutorials/</loc><lastmod>2020-08-06</lastmod><changefreq>weekly</changefreq><priority>0.9</priority></url><url><loc>https://proto.school/events/</loc><lastmod>2020-08-06</lastmod><changefreq>weekly</changefreq><priority>0.8</priority></url><url><loc>https://proto.school/news/</loc><lastmod>2020-08-06</lastmod><changefreq>monthly</changefreq><priority>0.7</priority></url><url><loc>https://proto.school/host/</loc><lastmod>2020-08-06</lastmod><changefreq>monthly</changefreq><priority>0.6</priority></url><url><loc>https://proto.school/build/</loc><lastmod>2020-08-06</lastmod><changefreq>monthly</changefreq><priority>0.6</priority></url><url><loc>https://proto.school/contribute/</loc><lastmod>2020-08-06</lastmod><changefreq>monthly</changefreq><priority>0.6</priority></url><url><loc>https://proto.school/data-structures/</loc><lastmod>2020-08-06</lastmod><changefreq>monthly</changefreq><priority>1.0</priority></url><url><loc>https://proto.school/basics/</loc><lastmod>2020-08-06</lastmod><changefreq>monthly</changefreq><priority>1.0</priority></url><url><loc>https://proto.school/blog/</loc><lastmod>2020-08-06</lastmod><changefreq>monthly</changefreq><priority>1.0</priority></url><url><loc>https://proto.school/mutable-file-system/</loc><lastmod>2020-08-06</lastmod><changefreq>monthly</changefreq><priority>1.0</priority></url><url><loc>https://proto.school/regular-files-api/</loc><lastmod>2020-08-06</lastmod><changefreq>monthly</changefreq><priority>1.0</priority></url><url><loc>https://proto.school/anatomy-of-a-cid/</loc><lastmod>2020-08-06</lastmod><changefreq>monthly</changefreq><priority>1.0</priority></url><url><loc>https://proto.school/verifying-storage-on-filecoin/</loc><lastmod>2020-08-06</lastmod><changefreq>monthly</changefreq><priority>1.0</priority></url></urlset>
```

Comment:
Any reason not to exclude this file from git and have it autogenerated during the build? It's normally a footgun to to have generated resources commited to the repo.

Reply:
Ah sounds good 👍 
Was thinking it would be nice to have a local committed version, but we can always run the build command to test the final result.

Reply:
Done!